src/components/TodoInfo.tsx

```diff
@@ -0,0 +1,22 @@
+// import Reat from 'react';
```

Comment:
why do you comment this line?
```suggestion
import React from 'react';
```